notebooks/NIRCam/NIRCam_claw_detection/nircam_claw_detection.ipynb

```diff
@@ -0,0 +1,461 @@
+    "<a id='exposure-level-plots'></a>\n",
+    "### Exposure Level Valid Angle Plots"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b1989436-251a-4b12-a612-3a5b86afccc4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "supported_observations = program.observations.supported_observations.keys()\n",
+    "for obs_number in supported_observations:\n",
+    "    program.plot_exposures(program.observations.data[obs_number])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7ad96131-f5aa-48a7-a37e-720f5a1ced00",
+   "metadata": {},
+   "source": [
+    "<a id='observation-level-plots'></a>\n",
+    "### Observation Level Valid Angle Plots\n",
+    "This will show valid angle ranges across all exposures in a single observation. If angles ranges are valid in one exposure but not in others, it will not be shown at this level."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a3483e33-2816-4921-9eb4-912b7a49c136",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "supported_observations = program.observations.supported_observations.keys()\n",
+    "for obs_number in supported_observations:\n",
+    "    program.plot_observation(program.observations.data[obs_number])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9d8f4b2d-c969-4e9b-b4ed-e5776184dbf6",
+   "metadata": {},
+   "source": [
+    "<a id='v2-vs-v3'></a>\n",
+    "### Plotting a Single Angle of Attitude in V2 & V3 Space\n",
+    "\n",
+    "The figure below shows what the tool is searching for when sweeping angles. Here for a single angle we show the V2 & V3 space of the targets in our source catalog as well as the susceptibility region. The intensity of targets that fall in this region are dependent on the location where of the target in the susceptibility region and the brightness of the target."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b4b4cda4-c003-4d9d-abf3-5758ba8e517c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from jwst_rogue_path_tool.plotting import plot_fixed_angle_regions\n",
+    "\n",
+    "angles = np.arange(100, 106)\n",
+    "\n",
+    "for angle in angles:\n",
+    "    plot_fixed_angle_regions(observation, angle)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b647f76c-b6c1-42db-a518-9add1195efd4",
+   "metadata": {},
+   "source": [
+    "<a id='attitude-vs-flux'></a>\n",
+    "### Flux as a Function of Attitude\n",
+    "\n",
+    "Below we show the calculated flux as a function of attitude angle. This figure also ties in the [JWST Backgrounds Tool](https://github.com/spacetelescope/jwst_backgrounds) to calculate the background of the target JWST is observering described in the APT file. By default, we set the parameters for the backgrounds tool to calculate 10% of the minimum background value. You can set the function and percentage via a dictionary and pass it to the AptProgram class.\n",
```

Comment:
typo "observering"

Reply:
Between the number of times I typed observation and observing, this was inevitable 😓